core/tests/protractor_utils/waitFor.js

```diff
@@ -131,6 +131,15 @@ var newTabToBeCreated = async function(errorMessage, urlToMatch) {
   }, DEFAULT_WAIT_TIME_MSECS, errorMessage);
 };
 
+/**
+ * @param {string} url - URL to redirect
+ */
+var urlRedirection = async function(url) {
+  // Checks that the current URL matches the expected text.
+  await browser.wait(until.urlIs(url), DEFAULT_WAIT_TIME_MSECS,
```

Comment:
Do we need `await` for `until.urlIs(url)`?

Reply:
Yes, I believe it is needed because it is an async function and would raise a promise error otherwise.